docs/pages/versions/unversioned/sdk/sqlite.mdx

```diff
@@ -193,6 +193,46 @@ try {
 }
 ```
 
+### Tagged template literals API
+
+For convenience and improved developer experience, expo-sqlite provides a Bun-inspired tagged template literals API through the `db.sql` property. This API automatically escapes parameters to prevent SQL injection attacks, and provides automatic type inference based on the query type.
+
```

Comment:
```suggestion
For convenience and improved developer experience, expo-sqlite provides Bun-inspired tagged template literals API through the `db.sql` property. This API automatically escapes parameters to prevent SQL injection attacks, and provides automatic type inference based on the query type.

```